### ISSUE TITLE
Yanking in the "name" field drops you into the Lisp REPL

### ISSUE THREAD

**Issue by rigdern** [boxer-project/boxer-sunrise#102]
# Repro steps
I reproed this with Boxer `3.4.19 2024-06-24` on macOS.

1. Type the text "abc"
2. Highlight "abc" with the mouse
3. Hit the delete key (it deletes "abc")
4. Create a data box by hitting the `{` key
5. Edit the name field by hitting the `|` key
6. Yank into the name field by hitting command-y

**Unexpected result:** I get dropped into the Common Lisp REPL in Terminal.app:

```
Error: No applicable methods for #<STANDARD-GENERIC-FUNCTION BOXER::SCREEN-BOX 80D125AF7C> with args (NIL)
  1 (continue) Call #<STANDARD-GENERIC-FUNCTION BOXER::SCREEN-BOX 80D125AF7C> again
  2 (abort) Return to Cocoa Event Loop

Type :b for backtrace or :c <option number> to proceed.
Type :bug-form "<subject>" for a bug report template or :? for other options.
```

When this happens, the only way I've found to get Boxer back into a good state is to restart it.

This doesn't repro when trying to paste "abc" rather than yank "abc".

**Comment by sgithens** [comment]
Fixed in 30e70c3 with another nil check, not great, but holding off a bit before doing the full dive into kill/yank behavior to start the undo/redo implementation. Will be in the next build.

**Comment by rigdern** [comment]
I confirmed this no longer repros in Boxer version `3.4.20 2024-08-06`
